integreat_cms/core/management/commands/find_database_inconsistencies.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
+
+import logging
+from argparse import ArgumentTypeError
+from typing import TYPE_CHECKING
+
+from django.db.models import F, OuterRef, Q, Subquery
+
+from ....cms.models import (
+    EventTranslation,
+    LanguageTreeNode,
+    Page,
+    PageTranslation,
+    POITranslation,
+)
+from ..log_command import LogCommand
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from django.core.management.base import CommandParser
+    from django.db.models.base import ModelBase
+
+logger = logging.getLogger(__name__)
+
+#: The possible translation model choices
+VERSION_MODELS: list[ModelBase] = {
+    PageTranslation,
+    EventTranslation,
+    POITranslation,
+}
+
+#: The possible tree model choices
+TREE_MODELS: list[ModelBase] = {
+    Page,
+    LanguageTreeNode,
+}
+
+
+class Command(LogCommand):
+    """
+    Management command to find missing versions
+    """
+
+    help = "Find version inconsistencies in the CMS"
+
+    def find_tree_inconsistencies(self):
+        """
+        Find inconsistencies in MPTT structure
+        """
+        for model in TREE_MODELS:
+            self.print_info(
+                f"Checking the model {model.__name__} for tree inconsistencies..."
+            )
+            inconsistencies = model.objects.filter(
+                Q(
+                    lft__in=Subquery(
+                        model.objects.filter(tree_id=OuterRef("tree_id")).values_list(
+                            "rgt", flat=True
+                        )
+                    )
+                )
+                | Q(
+                    rgt__in=Subquery(
+                        model.objects.filter(tree_id=OuterRef("tree_id")).values_list(
+                            "lft", flat=True
+                        )
+                    )
+                )
+            )
+            if len(inconsistencies):
+                self.print_error(f"Oh oh, there are tree inconsistencies!")
```

Comment:
*very important contribution*
```suggestion
                self.print_error(f"Uh oh, there are tree inconsistencies!")
```